api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/IgnoredParameters.scala

```diff
@@ -30,16 +30,14 @@ object IgnoredParameters {
     "calibration_frame", // GBM, DRF, XGBoost
     "max_hit_ratio_k", // GBM, DRF, Deep Learning
     "checkpoint", // GBM, DRF, XGBoost, Deep Learning
-    "plug_values", // GLM
     "interaction_pairs", // GLM
     "beta_constraints", // GLM
     "rand_link", // GLM
     "rand_family", // GLM
     "random_columns", // GLM
     "initial_biases", // DeepLearning
     "initial_weights", // DeepLearning
-    "pretrained_autoencoder", // DeepLearning
-    "build_control") //AutoML
```

Comment:
Is build control removed on purpose here?

Reply:
Yes, AutoML is handled in `AutoMLIgnoredParameters`

Reply:
Thanks! I checked https://github.com/h2oai/sparkling-water/blob/master/api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/AutoMLIgnoredParameters.scala and build_control is missing in the list, should we add it there?

Reply:
IMO, it won't have any effect since we generate each parameter section separately. https://github.com/h2oai/sparkling-water/blob/master/api-generation/src/main/scala/ai/h2o/sparkling/api/generation/Runner.scala#L112

`"build_control"` was added when i didn't understand how Automl parameters are structured, but forgot to remove it.

Reply:
Perfect, thanks for great explanation!